x/evm/types/keys.go

```diff
@@ -1,5 +1,9 @@
 package types
 
+var (
+	ParamsKey = []byte{0x00}
```

Comment:
Where are we using this?

Reply:
Previously i thought that evm module had params.So that's why i haved added it .I'll remove it